### PR TITLE
feat(pmp-web): add loading indicators to all views

### LINKED ISSUE #96: Add loading indicators to views

### PR DESCRIPTION
**Pull request by wjanaszek**
Add @ngneat/content-loader library to show facebook-like loading placeholders.

Resolves #96

### CODE REVIEW

File: libs/pmp-web/repository/repository-statistics/feature/src/lib/repository-statistics/repository-statistics.component.ts [hunk 1]
```diff
@@ -31,4 +38,14 @@ export class RepositoryStatisticsComponent implements OnDestroy, OnInit {
       this.facade.getRepositoryStatistics({ id: this.repositoryId });
     });
   }
+
+  private initRepositoryInfoFromNavigation(): void {
```
Comment (MaciejSikorski) [inline libs/pmp-web/repository/repository-statistics/feature/src/lib/repository-statistics/repository-statistics.component.ts]: From where is that change?

File: libs/pmp-web/user/users-statistics/feature/src/lib/containers/users-statistics/users-statistics.component.ts [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-  ngOnDestroy(): void {}
-
   ngOnInit(): void {
     this.userFacade.getUserStatisticsCollection({});
   }
 
   onNavigateToUser(userStatistics: UserStatistics): void {
-    this.router.navigate(['user', userStatistics.name]);
+    this.router.navigate(['user', userStatistics.name], {
+      state: { avatarUrl: userStatistics.avatarUrl }
```
Comment (MaciejSikorski) [inline libs/pmp-web/user/users-statistics/feature/src/lib/containers/users-statistics/users-statistics.component.ts]: what is that state?

File: libs/pmp-web/shared/ui-statistics-overview-table/src/lib/statistics-overview-table/statistics-overview-table.component.scss [hunk 3]
```diff
@@ -34,3 +34,9 @@ mat-cell {
   border-radius: 50%;
   margin: auto;
 }
+
+.content-loader {
```
Comment (MaciejSikorski) [inline libs/pmp-web/shared/ui-statistics-overview-table/src/lib/statistics-overview-table/statistics-overview-table.component.scss]: it shouldn't be duplicated

Comment (wjanaszek) [reply libs/pmp-web/shared/ui-statistics-overview-table/src/lib/statistics-overview-table/statistics-overview-table.component.scss]: https://angular.io/api/router/NavigationExtras#state

File: libs/pmp-web/user/users-statistics/feature/src/lib/containers/users-statistics/users-statistics.component.ts (hunk 2 above)
Comment (wjanaszek) [reply libs/pmp-web/user/users-statistics/feature/src/lib/containers/users-statistics/users-statistics.component.ts]: https://angular.io/api/router/NavigationExtras#state

File: libs/pmp-web/shared/ui-statistics-overview-table/src/lib/statistics-overview-table/statistics-overview-table.component.scss (hunk 3 above)
Comment (wjanaszek) [reply libs/pmp-web/shared/ui-statistics-overview-table/src/lib/statistics-overview-table/statistics-overview-table.component.scss]: So where should it be placed? In global assets directory?

Comment (MaciejSikorski) [reply libs/pmp-web/shared/ui-statistics-overview-table/src/lib/statistics-overview-table/statistics-overview-table.component.scss]: Yes, until we confirm or delegate to someone to implement that idea with styles.

**Pull request merged**